ssl/quic/quic_impl.c

```diff
@@ -2177,6 +2177,53 @@ struct quic_write_again_args {
     int                 err;
 };
 
+#define MAX_WRITE_BUF_SIZE      (6 * 1024 * 1024)
```

Comment:
This seems like a constant that should be placed somewhere in the internal headers although it is used only in this code.

Reply:
Do you have an existing header under `internal/quic_*` in mind? I do not think there is an obvious place for it. See also the top of quic_channel.c

Reply:
hmm... got it. This would not be an exception... I am ok with the current placement then for 3.2 at least. We can do some quic limits refactoring PR later for 3.3.